packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -0,0 +1,275 @@
+  },
+  render: (args: any) => {
+    return html`
+      ${['front', 'back', 'mediaFront', 'mediaBack'].map(slot => {
+        return generateTemplate({
+          axis: {
+            x: {
+              type: 'slot',
+              name: slot,
+              title: 'slot=..',
+              values: [
+                {
+                  value: `<div slot='${slot}' class="slot slot--border slot--background w-full h-full"></div>`,
+                  title: slot
+                }
+              ]
+            }
+          },
+          args,
+          constants: [
+            {
+              type: 'template',
+              name: 'style',
+              value: '<div style="margin-bottom: 40px; width: 300px">%TEMPLATE%</div>'
+            },
+            {
+              type: 'attribute',
+              name: 'frontVariant',
+              value: 'gradient-dark-top'
+            },
+            {
+              type: 'attribute',
+              name: 'backVariant',
+              value: 'gradient-dark-bottom'
+            }
+          ]
+        });
+      })}
+    `;
+  }
+};
+
+/**
+ * Use the `base`, `front`, `back`, `frontSlotContainer`, `backSlotContainer`, `mediaFront`, `mediaBack`, `frontSecondaryGradient` and `backSecondaryGradient` parts to style the flipcard.
+ */
+export const Parts = {
+  parameters: {
+    controls: {
+      exclude: [
+        'base',
+        'front',
+        'back',
+        'frontSlotContainer',
+        'backSlotContainer',
+        'mediaFront',
+        'mediaBack',
+        'frontSecondaryGradient',
+        'backSecondaryGradient'
+      ]
+    }
+  },
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'template',
+          name: 'sd-flipcard::part(...){outline: solid 2px red}',
+          values: [
+            'base',
+            'front',
+            'back',
+            'frontSlotContainer',
+            'backSlotContainer',
+            'mediaFront',
+            'mediaBack',
+            'frontSecondaryGradient',
+            'backSecondaryGradient'
+          ].map(part => {
+            return {
+              title: part,
+              value: `<style>#part-${part} sd-flipcard::part(${part}){outline: solid 2px red; outline-offset: -2px;}</style><div id="part-${part}">%TEMPLATE%</div>`
+            };
+          })
+        }
+      },
+      args,
+      constants: [
+        {
+          type: 'template',
+          name: 'style',
+          value: '<div style="margin-bottom: 40px; width: 300px">%TEMPLATE%</div>'
+        },
+        {
+          type: 'attribute',
+          name: 'frontVariant',
+          value: 'gradient-dark-top'
+        },
+        {
+          type: 'attribute',
+          name: 'darkVariant',
+          value: 'gradient-dark-bottom'
+        }
+      ]
+    });
+  }
+};
+
+/**
+ * `sd-flipcard` is fully accessibile via keyboard.
+ */
+
+export const Mouseless = {
```

Comment:
is it possible to prevent imidiate switch, when opening the story? Its most probably as we change the focus on story-load but this feels buggy as well.